pages/components/Logo/getDimensions.ts

```diff
@@ -0,0 +1,62 @@
+type dimension = string | undefined;
+type dimensions = [string, string];
+
+export const aspectRatio = 926 / 1065;
+
+function validateMeasurement(str: string | undefined): Boolean {
+  const measurementRegex = /^\d+\.?\d*[a-zA-Z%]+$/;
+
+  if (str === undefined) {
+    return false;
```

Comment:
I am torn here - should `undefined` be considered an invalid "measurement" if we do not want to throw? Also, the responsibility of throwing an error on an invalid measurement should not belong to the validation function. The validation function should only return a `true`/`false` response.

Reply:
I have refactored the method so that `isMeasurementValid` returns a `true`/`false` without throwing. The logic to throw is moved to a helper function `throwMeasurementValidation` called by the exported function `getDimensions` should either dimension fail validation.